src/ops/cli/helmfile.py

```diff
@@ -93,8 +93,19 @@ def get_tmp_file():
     def generate_helmfile_config(self, path, args):
         output_file = args.helmfile_path + "/hiera-generated.yaml"
         logger.info('Generating helmfiles config %s', output_file)
+
+        excluded_keys = []
+        filtered_keys = []
+
+        if "helmfile" in self.ops_config["compositions"]["excluded_config_keys"]:
```

Comment:
What happens if excluded_config_keys/filtered_output_keys are missing from the `.opsconfig`? I think this will crash, which is not ideal (some people might not have the .opsconfig file).

Based on this https://docs.python.org/2/glossary.html#term-eafp, what do you think if we do something like:

```py
try:
    excluded_keys += self.ops_config["compositions"]["excluded_config_keys"]["helmfile"]
except KeyError:
    excluded_keys = []
```